src/components/Authorization/Authorization.jsx

```diff
@@ -1,145 +1,237 @@
+        >
+          <img
+            src={imageCard}
+            alt="картинка с сине-голубым оттенком"
+            className="imageCard"
+          />
+          <div className="dialog-window">
+            <h1 className="dialog-title">Mы рады вас видеть</h1>
+
             <form className="dialog-content" onSubmit={handleSubmit}>
-              <p className={"error " + (emailError ? "error-text" : "")}>{emailError}</p>
-              <input className='input-field' type="email"
-                          id="email"
-                          name="email"
-                          placeholder="Логин"
-                          value={email}
-                          onChange={onEmailChange}
-                          style={{borderColor: emailError ? "red" : "##282828", borderWidth: '1px'}}/>
-              <p className={"error " + (passwordError ? "error-text" : "")}>{passwordError}</p>
-              <input className='input-field' type={passwordVisible ? 'text' : 'password'}
-                          id="pass"
-                          name="password"
-                          minLength={8}
-                          required
-                          value={password}
-                          placeholder="Пароль"
-                          onChange={onPasswordChange}
-                          style={{borderColor: passwordError ? "red" : "#282828", borderWidth: '1px'}}/>
+              {emailError && (
+                <p className={'error ' + 'error-text'}>{emailError}</p>
+              )}
+              <div
+                className={`form-floating mb-3 input-field ${email ? 'has-value' : ''}`}
+              >
+                <input
+                  className={`form-control ${emailError ? 'is-invalid' : ''}`}
+                  id="floatingInput"
+                  type="email"
+                  name="email"
+                  placeholder=" "
+                  value={email}
+                  onChange={onEmailChange}
+                />
+                <label htmlFor="floatingInput">Почта</label>
+              </div>
+              {passwordError && (
+                <p className={'error ' + 'error-text'}>{passwordError}</p>
+              )}
+              <div
+                className={`form-floating mb-3 input-field ${password ? 'has-value' : ''}`}
+              >
+                <input
+                  className={`form-control ${passwordError ? 'is-invalid' : ''}`}
+                  type={passwordVisible ? 'text' : 'password'}
+                  id="pass"
+                  name="password"
+                  minLength={8}
+                  required
+                  value={password}
+                  placeholder=" "
+                  onChange={onPasswordChange}
+                />
+                <label className="lab" htmlFor="pass">
+                  Пароль
+                </label>
+
+                {passwordVisible ? (
+                  <svg
```

Comment:
Use <svg> icon in main file bag idea. Icon should be isolated one file

Reply:
fix